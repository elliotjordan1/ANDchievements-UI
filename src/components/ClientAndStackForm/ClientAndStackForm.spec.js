import React from 'react';
import { render } from '@testing-library/react';
import TechStackForm from './ClientAndStackForm';

describe('Form Modal component', () => {
  it('renders correctly for a client', () => {
    const wrapper = render(<TechStackForm name='Test Stack' isClient />);
    expect(wrapper).toMatchSnapshot();
  });

  it('renders correctly for a Tech Stack', () => {
    const wrapper = render(<TechStackForm name='Test Stack' />);
    expect(wrapper).toMatchSnapshot();
  });
});
