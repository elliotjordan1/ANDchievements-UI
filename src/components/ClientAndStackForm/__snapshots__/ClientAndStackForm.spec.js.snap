// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form Modal component renders correctly for a Tech Stack 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <form
        class="styles__FormWrapper-sc-1o7k5k1-1 iFktw"
      >
        <div
          class="styles__Label-sc-1o7k5k1-0 loicrU"
        >
          Test Stack
        </div>
        <div>
          <div
            class="styles__Label-sc-1o7k5k1-0 loicrU"
          >
            Tech Stack
             Image Url
          </div>
          <input
            class="styles__FormInput-sc-1o7k5k1-2 kHKDTI"
            placeholder="Tech Stack image url"
            value=""
          />
        </div>
        <div>
          <button
            class="styles__SubmitButton-sc-1o7k5k1-3 jToJUi"
            type="submit"
          >
            SUBMIT
          </button>
        </div>
      </form>
    </div>
  </body>,
  "container": <div>
    <form
      class="styles__FormWrapper-sc-1o7k5k1-1 iFktw"
    >
      <div
        class="styles__Label-sc-1o7k5k1-0 loicrU"
      >
        Test Stack
      </div>
      <div>
        <div
          class="styles__Label-sc-1o7k5k1-0 loicrU"
        >
          Tech Stack
           Image Url
        </div>
        <input
          class="styles__FormInput-sc-1o7k5k1-2 kHKDTI"
          placeholder="Tech Stack image url"
          value=""
        />
      </div>
      <div>
        <button
          class="styles__SubmitButton-sc-1o7k5k1-3 jToJUi"
          type="submit"
        >
          SUBMIT
        </button>
      </div>
    </form>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Form Modal component renders correctly for a client 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <form
        class="styles__FormWrapper-sc-1o7k5k1-1 iFktw"
      >
        <div
          class="styles__Label-sc-1o7k5k1-0 loicrU"
        >
          Test Stack
        </div>
        <div>
          <div
            class="styles__Label-sc-1o7k5k1-0 loicrU"
          >
            Client
             Image Url
          </div>
          <input
            class="styles__FormInput-sc-1o7k5k1-2 kHKDTI"
            placeholder="Client image url"
            value=""
          />
        </div>
        <div>
          <button
            class="styles__SubmitButton-sc-1o7k5k1-3 jToJUi"
            type="submit"
          >
            SUBMIT
          </button>
        </div>
      </form>
    </div>
  </body>,
  "container": <div>
    <form
      class="styles__FormWrapper-sc-1o7k5k1-1 iFktw"
    >
      <div
        class="styles__Label-sc-1o7k5k1-0 loicrU"
      >
        Test Stack
      </div>
      <div>
        <div
          class="styles__Label-sc-1o7k5k1-0 loicrU"
        >
          Client
           Image Url
        </div>
        <input
          class="styles__FormInput-sc-1o7k5k1-2 kHKDTI"
          placeholder="Client image url"
          value=""
        />
      </div>
      <div>
        <button
          class="styles__SubmitButton-sc-1o7k5k1-3 jToJUi"
          type="submit"
        >
          SUBMIT
        </button>
      </div>
    </form>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
